shared/src/store/action-reducers/exercise.ts

```diff
@@ -151,3 +215,26 @@ function refreshTransfer(
         letElementArrive(draftState, key, element.id);
     });
 }
+
+/**
+ * Prepare a {@link PartialExport} for import.
+ *
+ * This includes resetting UUIDs as this cannot be done in the reducer.
+ * @param partialExport The {@link PartialExport} to prepare.
+ */
+export function preparePartialExportForImport(
+    partialExport: PartialExport
+): PartialExport {
+    const copy = cloneDeepMutable(partialExport);
+    // `patientCategories` don't have an `id`
+    const properties = ['mapImageTemplates', 'vehicleTemplates'] as const;
+    for (const property of properties) {
+        const value = copy[property];
+        if (value !== undefined) {
+            for (const entry of value) {
+                entry.id = uuid();
+            }
+        }
+    }
```

Comment:
This would be easier to understand, if instead of `property`, `value` and `entry` this would be `templateType`, `templates`, and `template`.

YAGNI